### PR TITLE
PD-285   Write tutorial for Google Photos credentials created with Google API

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- scalebackup
+---
+
+{{< toc >}}
+
+Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+
+## Creating the API Credentials
+
+On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu next to the Google Cloud logo and select your project. If you do not have a project, click **NEW PROJECT** and enter a **Project name**, **Organization**, and **Location**.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:17]: small change suggestion. As a style suggestion (from most of the style guides out there), we don't want to use the UI elements in this way so I suggest this:
```suggestion
On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu next to the Google Cloud logo and select your project. If you do not have a project, click **NEW PROJECT** and enter a value in **Project name**, **Organization**, and **Location**.
```

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+![GooglePhotosAPINewProject](/images/SCALE/22.12/GooglePhotosAPINewProject.png "New Project")
+
+### Enable API
+
+After you select your project, click **Enabled APIs & Services** on the left menu, then click **+ ENABLE APIS AND SERVICES**.
+
+![GooglePhotosAPIEnableAPIsandServices](/images/SCALE/22.12/GooglePhotosAPIEnableAPIsandServices.png "Enable APIs and Services")
+
+Enter *photos library api* in the search bar, then select **Photos Library API** and click **ENABLE**.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:27]: is this a variable or what they actually enter in the search field? I know it is a lot of bold, but if it is specific we use bold for those values. If this is just one way to search for it, italics is okay but clarify this is a way to search for it.
```suggestion
Enter *photos library api* in the search bar, then select **Photos Library API** and click **ENABLE**.
```

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+![GooglePhotosAPIClickEnable](/images/SCALE/22.12/GooglePhotosAPIClickEnable.png "Click Enable")
+
+### Configure Authentication
+
+Next, click **OAuth consent screen** on the left menu, select **EXTERNAL**, then click **CREATE**.
+
+![GooglePhotosAPICreateExternal](/images/SCALE/22.12/GooglePhotosAPICreateExternal.png "Create External")
+
+Enter an **App name** and **User support email**.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:39]: Same reason as above. It is best to not use the field name as the "value" to enter.
```suggestion
Enter a value in **App name** and **User support email**. 
```

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+Continue to the **Add users** section, enter your email address, then click **ADD**.
+
+![GooglePhotosAPIAddTestingUser](/images/SCALE/22.12/GooglePhotosAPIAddTestingUser.png "Add Testing User")
+
+On the **OAuth consent screen**, click **PUBLISH APP** under **Testing** and push the app to production.
+
+![GooglePhotosAPIPublish](/images/SCALE/22.12/GooglePhotosAPIPublish.png "Publish Status")
+
+{{< expand "Can I leave the app in testing mode?" "v" >}}
+You can leave the app in testing mode, but your cloud sync task will fail after your testing app credentials expire after seven days.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:56]: active voice, present tense suggestion
```suggestion
You can leave the app in testing mode, but your cloud sync task fails after your testing app credentials expire after seven days. 
```

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+Enter a name for the new remote, then enter the number from the list corresponding to Google Photos.
+
+Enter the client id and secret you saved when you created the Google Photos API credentials, then enter `false` to keep the Google Photos backend read-only.
+
+![GooglePhotosAPIrcloneConfig2](/images/SCALE/22.12/GooglePhotosAPIrcloneConfig2.png "Configure rclone")
+
+Do not edit the advanced config. When prompted about automatically authenticating rclone with the remote, enter `y`.
+
+A browser window opens to authorize rclone access. Click **Allow*.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:89]:
```suggestion
A browser window opens to authorize rclone access. Click **Allow**.
```

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+In the command line, enter `y` when prompted about media item resolution to complete the configuration.
+
+Copy and save the Options (type, client_id, client_secret, token), then enter `y` to keep the new remote.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:93]: is Options something on the screen? if yes use bold, if not or a general reference to options use lower case

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Just a few things related to style. Excellent article!

File: content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+![GooglePhotosAPIrcloneConfig2](/images/SCALE/22.12/GooglePhotosAPIrcloneConfig2.png "Configure rclone")
+
+Do not edit the advanced config. When prompted about automatically authenticating rclone with the remote, enter `y`.
+
+A browser window opens to authorize rclone access. Click **Allow**.
+
+In the command line, enter `y` when prompted about media item resolution to complete the configuration.
+
+Copy and save the Options (type, client_id, client_secret, token), then enter `y` to keep the new remote.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md:93]:
```suggestion
Copy and save the type, client_id, client_secret, and token, then enter `y` to keep the new remote.
```

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**